www/js/controllers/menu_controller.js

```diff
@@ -0,0 +1,38 @@
+app.controller('MenuCtrl', function($scope, $state, $ionicLoading, $ionicPopup, BarcodeService, UserService, $ionicHistory) {
+
+  // Logout
+
+  $scope.logout = function(){
+    $ionicLoading.show();
+
+    UserService.logout()
+    .then(function(str){
```

Comment:
The logout function's promise does not resolve with a string. So the `str` param is not necessary here.